magento/magento2
# Removed unnecessary spacing on backend grid

### Description (*)

This PR fixes minor issue on backend grid. When there are selected items, there is unnecessary spacing being displayed.
Main change is line 68 - rest is only refactoring.
![image](https://user-images.githubusercontent.com/9566840/112909133-24920700-90f1-11eb-9904-6cdcb24f2e9d.png)


### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. In Backend, go to System -> Tools -> Cache Management
2. Select few rows on grid
3. Make sure there is no spacing in **(X selected)** phrase
![image](https://user-images.githubusercontent.com/9566840/112909328-7b97dc00-90f1-11eb-948d-ab610b56e098.png)


### Questions or comments

To reproduce this issue, HTML minification probably must be disabled.
I don't think any tests are applicable here.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38502: Removed unnecessary spacing on backend grid
